controllers/topics.controller.js

```diff
@@ -0,0 +1,11 @@
+const { catchAsync } = require("../utils/utils");
+const { fetchTopics } = require("../models/topics.model");
+
+exports.getTopics = catchAsync(async (req, res, next) => {
+  const topics = await fetchTopics();
+
+  return res.status(200).json({
+    status: "success",
+    topics,
+  });
```

Comment:
As you've already set up error handling middleware, dont forget your .catch block